src/cheri/rvycheriot-encoding.adoc

```diff
@@ -0,0 +1,359 @@
+| CT         | Sealed object Type, see <<section_cheriot_cap_type>>.
+| E          | Exponent field for bounds encoding, see <<section_cheriot_cap_bounds>>.
+| T          | Top field for bounds encoding, see <<section_cheriot_cap_bounds>>.
+| B          | Base field for bounds encoding, see <<section_cheriot_cap_bounds>>.
+| Address    | The address field as per <<cap_structure>>.
+|==============================================================================
+
+The encoding impacts the following CHERI extensions:
+
+<<section_cheri_hybrid_ext>>::
+This encoding does not support {cheri_default_ext_name}: there is no mode bit so only purecap mode is supported.
+
+<<section_zylevels1>>::
+This encoding supports <<section_zylevels1>>.
+The <<zylevels1_gl_perm,GL>> flag is encoded in the G bit.
+
+<<section_zyseal>>::
+This encoding supports <<section_zyseal>>.
+The <<zyseal_se_perm,SE>> and <<zyseal_us_perm,US>> permissions are encoded in the P field.
+
+This capability encoding has the following properties that affect the observable behavior of {cheri_base_ext_name} instructions such as <<SCBNDS>> and <<CLRPERM>>:
+
+* **Mantissa width (mw{cap_cheriot_mw_width})**: The mantissa width for the bounds encoding uses {cap_cheriot_mw_width} bits
+* **Maximum exponent (e{cap_cheriot_exp_max})**: The maximum value for the exponent in a valid capability is {cap_cheriot_exp_max}, however exponents 15 to 23 are not supported (see below).
+* **Representable region (rc0)**: Only addresses between encoded base and top (inclusive) are guaranteed to be representable.
+* **Permission encoding (pc)**: The permissions are encoded using a compressed format that cannot represent every combination of permissions.
+
+==== Capability Encoding Parameter Summary
+
+.{rvy32e_cheriot_base_name} parameter summary
+[#rvy32e_cheriot_base_name_param_summary,width="100%",options=header,]
+|==============================================================================
+| Parameter | Value                | Comment
+| mw        | {cap_cheriot_mw_width}  | Mantissa width
+| e         | 14                   | Exponent limit before jumping to maximum exponent
+| rc        | 0                    | Representable region between base and top only
+| p         | pc                   | Compressed permission encoding
+| as        | 11                   | <<AUIPC>> shift
+|==============================================================================
+
+.{rvy32e_cheriot_base_name} extension summary
+[#rvy32e_cheriot_base_name_ext_summary,width="100%",options=header,]
+|==============================================================================
+| Extension                    | Comment
+| <<section_cheri_hybrid_ext>> | Not supported
+| <<section_zylevels1>>        | Compatible
+ifdef::cheri_standalone_spec[]
+| <<section_zyseal>>           | Compatible
+endif::[]
+| All RVY versions of other standard extensions | Compatible if the extension is compatible with RV32I
```

Comment:
```suggestion
| All RVY versions of other standard extensions | Compatible if the extension is compatible with RV32E
```

this format is explicitly targeting RV32E so this seems better?

Reply:
This is an interesting one. There's nothing preventing CHERIoT from working with RV32I we just chose E to reduce context size.

Reply:
I would say that you should change the prefix to RV32Y from RV32YE and just note that this family of encoding formats is recommended for RV32YE systems, and leave it there.